packages/c3/tests/unit/chart-builders/dimension-test.ts

```diff
@@ -33,7 +34,8 @@ const DATA = NaviFactResponse.create({
   ]
 });
 // prettier-ignore
-const DATA2 = NaviFactResponse.create({
+//@ts-ignore - skip injector
```

Comment:
why skip it?

Reply:
This was skipped since it was in a scope that it couldn't do a service lookup. I've moved it inside the test so it now passes it in